### DISCUSSION TITLE
Rabby wallet not detected

### DISCUSSION THREAD

**Discussion by RomualdH** [WalletConnect/walletconnect-monorepo#2618] [Developer Support]
Hi there!

I'm having only Rabby wallet enabled on the my browser,
But I can't connect when clicking the icon, I need to connect with the MetaMask icon.
Seems Rabby is considered a random injected wallet, even by prioritizing it via `explorerRecommendedWalletIds` option.

Thank you in advance,
Romuald

**Comment by xzilja** [comment]
I think their team just submitted details about their extension to our [cloud](https://cloud.walletconnect.com) so their icon should be shown once approved. You see metamask icon because rabby extension tries to "mask" ask metamask with setting `window.ethereum.isMetamask` option which we use as fallback.
